stl/inc/ranges

```diff
@@ -704,16 +704,19 @@ namespace ranges {
         /* [[no_unique_address]] */ _Copyable_box<_Ty> _Val{};
     };
 
+    template <class _Ty>
+    single_view(_Ty) -> single_view<_Ty>;
+
     namespace views {
         // VARIABLE views::single
         struct _Single_fn {
             // clang-format off
             template <class _Ty>
             _NODISCARD constexpr auto operator()(_Ty&& _Val) const noexcept(
-                noexcept(single_view{_STD forward<_Ty>(_Val)})) requires requires {
-                single_view{static_cast<_Ty&&>(_Val)};
+                noexcept(single_view<decay_t<_Ty>(_STD forward<_Ty>(_Val)))) requires requires {

```

Comment:
Tests are failing with:
```
D:\build\out\inc\ranges(716): error C2187: syntax error: ')' was unexpected here
```
because there's a missing angle bracket here.

Reply:
"Oops - I missed a line. Certainly I can fix one line without rerunning tests."